test/byContract/IexecAccessors/IexecAccessors.test.ts

```diff
@@ -276,8 +278,14 @@ describe('IexecAccessors', async () => {
     });
 
     it('eip712domainSeparator', async function () {
-        expect(await iexecPoco.eip712domain_separator()).to.equal(
-            '0xfc2178d8b8300e657cb9f8b5a4d1957174cf1392e294f3575b82a9cea1da1c4b',
+        expect(await iexecPoco.eip712domain_separator()).equal(
+            await hashDomain({
+                name: 'iExecODB',
```

Comment:
```suggestion
                // TODO use IexecWrapper.getDomain() (with some modifications).
                name: 'iExecODB',
```

Reply:
Not sure, but suggestion applied :) (eventually I would like to remove `hashDomain` call, to avoid needing to test our tools at some point) !